apps/__init__.py

```diff
@@ -1,9 +1,9 @@
-from android.firefox import AndroidFirefox
+from android.firefox import GeckoViewExampleApp, FenixApp, FennecApp, RefbrowApp
```

Comment:
Let's rename these to `GeckoViewExample`, `Fenix`, `Fennec`, and `ReferenceBrowser` (drop the `App` part)